course-materials/lab-instructions/lab-06/lab-06-ugly-charts.Rmd

```diff
@@ -48,35 +48,96 @@ use_git_config(user.name = "Your Name",
 
 This is the second week you're working in teams, so we're going to make things a little more interesting and let all of you make changes and push those changes to your team repository. Sometimes things will go swimmingly, and sometimes you'll run into merge conflicts. So our first task today is to walk you through a merge conflict!
 
-## Merge conflicts
 
-- When two collaborators make changes to a file and push the file to their repo, git merges these two files.
-- If these two files have conflicting content on the same line, git will produce a **merge conflict**.
+## Merges and merge conflicts
 
-### Set up
+- Pushing a repo replaces the code on GitHub with the code you have on your computer. 
```

Comment:
```suggestion
- Pushing to a repo replaces the code on GitHub with the code you have on your computer. 
```

Reply:
Also, the concept of "your computer" is a bit fuzzy when working on the cloud but I think this is ok / not confusing even though technically not accurate. I go back and forth about accuracy around this.

Reply:
Maybe the missing concept is that you have a working repo (on your computer or on RStudio or whatever) that you can commit to directly but you update the shared repo on GitHub by pushing.